Python/Python.sublime-syntax

```diff
@@ -228,63 +228,63 @@ contexts:
         2: keyword.control.flow.with.python
       push: with-body
     # except ... as ...:
-    - match: \b(except)\b
-      scope: keyword.control.flow.except.python
+    - match: \bexcept\b
+      scope: keyword.control.exception.except.python
       push:
-        - meta_scope: meta.statement.except.python
+        - meta_scope: meta.statement.exception.except.python
         - include: line-continuation-or-pop
         - match: ':'
-          scope: punctuation.section.block.except.python
+          scope: punctuation.section.block.exception.except.python
           pop: true
-        - match: '\b(as)\b'
-          scope: keyword.control.flow.as.python
+        - match: '\bas\b'
+          scope: keyword.control.exception.except.as.python
           set:
-            - meta_content_scope: meta.statement.except.python
+            - meta_content_scope: meta.statement.exception.except.python
             - include: line-continuation-or-pop
             - match: ':'
-              scope: meta.statement.except.python punctuation.section.block.except.python
+              scope: meta.statement.exception.except.python punctuation.section.block.exception.except.python
               pop: true
             - include: name
         - include: target-list
     - match: \bif\b
-      scope: keyword.control.flow.conditional.python
+      scope: keyword.control.conditional.if.python
       push:
-        - meta_scope: meta.statement.if.python
+        - meta_scope: meta.statement.conditional.python
```

Comment:
Why have `meta.statement.exception.catch` but not use `meta.statement.conditional.if` here?

Reply:
Because the keyword used to be `keyword.control.flow.conditional.python`. 

This raises the general question: Do we want to apply general scopes like `conditional`, `exception` to meta's at all?

Reply:
I'm indifferent on that tbh. I don't see myself using `meta.statement` for anything, really. I just added the scopes because it makes debugging easier and we already push into the context.

Reply:
Me, too. I can see only one use-case which is related with the punctuations, which would basically require a 3-level-meta-scope only.

One post in #1228 requests to be able to scope the punctuation with the same color as the keyword or based on control-type.

Instead of scoping 

    punctuation.section.block.conditional.if
    punctuation.section.block.conditional.else
    ...
    punctuation.section.block.loop
    ...

the meta could propably be used to distinguish punctuation types

    meta.statement.conditional punctuation.section.block.python
    meta.statement.exception punctuation.section.block.python
    meta.statement.loop punctuation.section.block.python

The second solution would help to keep scope names shorter and more general.